_includes/configuration.md

```diff
@@ -1,14 +1,14 @@
 ## Configuring {{ site.data.product.title_short }}
 
-After installing {{ site.data.product.title_short }} and running it for the first
+After installing {{ site.data.product.title_short }} and run it for the first
```

Comment:
I think running is more grammatically correct.

Reply:
"parallelism" with `installing`

Reply:
```
After installing and running it for the first time, some basic configuration is required.
You must complete the following steps:
```

Reply:
Updated.